### PR TITLE
Update khiepleh-mfe's Octocat Caption

### PR DESCRIPTION
**Pull request by khiepleh-mfe**
Progress toward #20

### CODE REVIEW

File: _posts/0000-01-09-khiepleh-mfe.md [hunk 1]
```diff
@@ -3,7 +3,10 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (bkentel-mcafee) [inline _posts/0000-01-09-khiepleh-mfe.md]: You forgot to change the title

Comment (khiepleh-mfe) [reply _posts/0000-01-09-khiepleh-mfe.md]: And my history is already a trashbin 😱

**Comment by bkentel-mcafee** [comment]
You should probably also reference the relevant issue that this change addresses.

File: _posts/0000-01-09-khiepleh-mfe.md (hunk 1 above)
Comment (lost-grail) [reply _posts/0000-01-09-khiepleh-mfe.md]:
```suggestion
title: "Look in the garage before it smells too bad!"
```

Comment (khiepleh-mfe) [reply _posts/0000-01-09-khiepleh-mfe.md]: I over-eagerly did my own update. Womp.

**Comment by khiepleh-mfe** [comment]
Look at all this collaboration! We're a shining beacon of hope for this repo.

**Pull request merged**